blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderPipelineImpl.java

```diff
@@ -33,6 +32,11 @@ public OrganizationFolderPipelineImpl(OrganizationFolder folder, Link parent) {
 
     }
 
+    @Exported
+    public String getIcon() {
+        return folder.getIcon().getImageOf("32x32");
```

Comment:
Actually I did some testing and this doesn't appear to be the right icon. Perhaps Stephen knows how to get the org avatar? I couldn't figure it out